scrypto/build.sbt

```diff
@@ -0,0 +1,26 @@
+import sbt.Keys.{homepage, scalaVersion}
+
+name := "scrypto"
+description := "Cryptographic primitives for Scala"
+
+javacOptions ++=
+  "-source" :: "1.8" ::
+    "-target" :: "1.8" ::
+    Nil
+
+libraryDependencies ++= Seq(
+  "org.rudogma" %% "supertagged" % "1.5",
+  "com.google.guava" % "guava" % "23.0",
+  "com.typesafe.scala-logging" %% "scala-logging" % "3.9.2",
+  "org.whispersystems" % "curve25519-java" % "0.5.0",
+  "org.bouncycastle" % "bcprov-jdk15to18" % "1.66"
```

Comment:
let's use the most recent version and check later that SDK use the same one.
https://search.maven.org/artifact/org.bouncycastle/bcprov-jdk15on/1.70/jar
check on SDK side:
```
        <!-- NOTE: this comes from scrypto and contains a vulnerability, we use bcprov-jdk15on:1.69 below,
         this will be removed with merging scrypto into sparkz-core -->
        <exclusion>
          <artifactId>bcprov-jdk15to18</artifactId>
          <groupId>org.bouncycastle</groupId>
        </exclusion>
```